lib/response.js

```diff
@@ -455,17 +455,22 @@ internals.Response.prototype._prepare = function (next) {
 
     const onDone = (source) => {
 
-        if (source instanceof Error) {
-            return next(Boom.wrap(source));
-        }
+        // do not use Hoek.nextTick because it will not work as intended
```

Comment:
ahh yes, I realized this after I updated my comment. Thanks for pointing that out.